src/Uno.UI/UI/Xaml/Controls/ItemsControl/ItemsControl.cs

```diff
@@ -62,6 +62,32 @@ public partial class ItemsControl : Control, IItemsControl
 
 		internal ScrollViewer ScrollViewer { get; private set; }
 
+		/// <summary>
+		/// Stores materialized containers starting a given index, so that their
+		/// ItemsControl.IndexForContainerProperty can be updated after the collection changes.		
```

Comment:
This description doesn't sound accurate, you're 'storing' anything. The method name is also no longer accurate since you're not saving the containers.

Reply:
I forgot to review the summary